From d50a048d5b36587441a11e5a4f1d11dae300985f Mon Sep 17 00:00:00 2001
From: Samuel Holland <samuel@sholland.org>
Date: Mon, 17 Sep 2018 01:32:20 +0000
Subject: [PATCH] test/sysroot-crates-are-unstable: Fix test when rpath is
 disabled

Without this environment var, the test can't run rustc to find
the sysroot path.
---
 .../run-make-fulldeps/sysroot-crates-are-unstable/Makefile    | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/tests/run-make/sysroot-crates-are-unstable/Makefile b/tests/run-make/sysroot-crates-are-unstable/Makefile
index 1e267fb9576ba..30c33c5c13d20 100644
--- a/tests/run-make/sysroot-crates-are-unstable/Makefile
+++ b/tests/run-make/sysroot-crates-are-unstable/Makefile
@@ -1,2 +1,4 @@
+-include ../tools.mk
+
 all:
-	'$(PYTHON)' test.py
+	env '$(HOST_RPATH_ENV)' '$(PYTHON)' test.py

